android/app/src/main/java/co/ello/ElloApp/MainActivity.java

```diff
@@ -391,7 +402,35 @@ public void onActionClicked(Snackbar snackbar) {
         registerReceiver(pushReceivedReceiver, new IntentFilter(ElloPreferences.PUSH_RECEIVED));
     }
 
+    private void setupRatePrompt(){
+        AppRate.with(this)
+            .setInstallDays(7)
+            .setLaunchTimes(3)
+            .setRemindInterval(7)
+            .setShowLaterButton(true)
+            .setDebug(true)
+            .setOnClickButtonListener(new OnClickButtonListener() {
+                @Override
+                public void onClickButton(int which) {
+                    switch (which) {
+                        case -1: track("rate prompt user attempted to rate app");
+                            break;
+                        case -2: track("rate prompt user declined to rate app");
+                            break;
+                        case -3: track("rate prompt remind me later");
+                            break;
+                        default:
+                                break;
+                    }
+                }
+            })
+            .monitor();
 
+        Boolean showRateDialog = AppRate.showRateDialogIfMeetsConditions(this);
```

Comment:
I believe this just checks the series of flags that have set to determine if the prompt should be shown or not.